SciELO.js

```diff
@@ -114,10 +113,19 @@ function scrape(doc, url) {
 						.split(";")
 						.map(function(x) { return x.trim(); })
 						.map(function(y) { return y.charAt(0).toUpperCase() + y.slice(1); });
-	}
-
-		item.libraryCatalog = "SciELO"
+		}
+				item.libraryCatalog = "SciELO"
+		var domAbstract = ZU.xpath(doc, '/html/body/div[1]/div[2]/div[2]/div/a[1]');
+		for (let d in domAbstract) {
+			var lookupAbstract = domAbstract[d].href;//Z.debug(lookupAbstract)	
```

Comment:
You overwrite `lookupAbstract` on every iteration, so only the last `.href` is preserved.

Reply:
iteration is indeed not necessary. i changed it.